utils/generate_third_party_notice.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const checker = require('license-checker');
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+(async () => {
+  for (const project of ['playwright-core', 'playwright-test']) {
+    const lines = [];
+    lines.push(`microsoft/${project}
+
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+
+This project incorporates components from the projects listed below. The original copyright notices and the licenses under which Microsoft received such components are set forth below. Microsoft reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
+`);
+
+    const allPackages = {};
+    const projectDir = path.join(__dirname, '..', 'packages', project);
+    const bundlesDir = path.join(projectDir, 'bundles');
+    for (const bundle of fs.readdirSync(bundlesDir)) {
+      const dir = path.join(bundlesDir, bundle);
+      execSync('npm ci', { cwd: dir });
```

Comment:
Why this one? I see every bundle package.json has `npm ci` in every command.

Reply:
It was so that i could just run that one alone at any moment